src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -168,9 +168,11 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   useEffect(() => {
     // in the event the saved selection not found.. will always fall back to the first idType in array
     setSelectedIdType(
-      selectionArray.some(
-        (selection) => selection.label === selectedIdType.label
-      )
+      selectionArray.some((selection) => {
+        return selection.label && selectedIdType.label
+          ? selection.label === selectedIdType.label
+          : false;
```

Comment:
Can we do `selection.label && selectedIdType.label && selection.label === selectedIdType.label` instead?

Reply:
This is done!